modules/util/color.go

```diff
@@ -4,22 +4,12 @@ package util
 
 import (
 	"fmt"
-	"math"
 	"strconv"
 	"strings"
 )
 
 // Check similar implementation in web_src/js/utils/color.js and keep synchronization
 
-// Return R, G, B values defined in reletive luminance
-func getLuminanceRGB(channel float64) float64 {
-	sRGB := channel / 255
-	if sRGB <= 0.03928 {
-		return sRGB / 12.92
-	}
-	return math.Pow((sRGB+0.055)/1.055, 2.4)
```

Comment:
This line was simply removed now?

Reply:
Yes, I think this function was causing the calculation to go wrong. As I understand it, hex colors are already in SRGB space and need no convertion.

Reply:
So I'm not color math expert, but I do know that the previous calculation was wrong and the current one with the removal of this function fixes it, so likely it's fine to remove.